engine/config/1.0/schema.json

```diff
@@ -634,9 +634,44 @@
             "required": ["id", "workers"],
             "additionalProperties": true
           }
+        },
+        "default_toolkit_type": {
+          "type": "string",
+          "enum": ["standard", "convenience", "premium"],
+          "description": "DEPRECATED: The default toolkit type to use for tool calls.",
+          "deprecated": true
+        },
+        "toolkits": {
+          "type": "array",
+          "description": "DEPRECATED: A list of toolkit configurations.",
+          "deprecated": true,
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The unique ID of the toolkit."
+              },
+              "type": {
+                "type": "string",
+                "enum": ["standard", "convenience", "premium"],
+                "description": "The type of the toolkit."
+              },
+              "aliases": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "description": "Aliases for the toolkit."
+              }
+            },
+            "required": ["id", "type"],
+            "additionalProperties": false
+          },
+          "additionalProperties": false
         }
       },
-      "required": ["directors"],
+      "required": ["directors", "default_toolkit_type"],
```

Comment:
Don't make this required -- otherwise, the file will lint as incorrect for a self-hosted customer who is editing `engine.yaml`. Instead, assume a default value for `default_toolkit_type` in the engine if the field is missing.